params/protocol_params.go

```diff
@@ -156,3 +157,5 @@ const (
 	// We charge for reading the balance, 1 debit, and 3 credits (refunding gas, paying the gas fee recipient, sending to the infrastructure fund)
 	AdditionalGasForNonGoldCurrencies uint64 = 3*ExpectedGasForCreditToTransactions + ExpectedGasForDebitFromTransactions + ExpectedGasToReadErc20Balance
 )
+
+const ContractCommGas uint64 = 2000000 // gas amount used for querying system contracts
```

Comment:
I don't think this is entirely accurate, as if you look above we specify different amounts for different calls to system contracts. Instead, consider specifying one specific to this call, e.g. `MaxGasForGetMinimumClientVersion`